packages/svelte/src/dom.ts

```diff
@@ -0,0 +1,253 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+import type { FormEncType, FormMethod } from "@remix-run/router";
```

Comment:
Just FYI - eventually this will all come from `@remix-run/router`, it's just not exported at the moment.  So I would avoid making any changes in here as you keep moving on svelte.  If you find bugs or need to change anything - we'll want to note that and make the corresponding change to the file currently in `react-router-dom`.  Hopefully we get the exports reworked in the 6.4 release in the next couple weeks though.